development-guide.md

```diff
@@ -173,16 +174,16 @@ In addition to the suggestions above, consider the following as well:
     - [File](https://fivetran.com/docs/files) (S3, Google Drive, etc)
     - [Fivetran Platform connector](https://fivetran.com/docs/logs/fivetran-platform)
 - Exercise `AlterTable` in various ways:
-    - Adding one or more column(s)
+    - Adding one or more columns
     - Change of primary key columns (adding and removing columns from primary key constraint)
-    - Changing data type of non-primary key column(s) 
-    - Changing data type of primary key column(s)
+    - Changing data type of non-primary key columns 
+    - Changing data type of primary key columns
 - Test tables with and without primary-key
  
 ## FAQ
 
 ### Is it possible for me to see the connector log output?
-Sort of. We will email you the logs for a failed sync through support but the number of log messages is limited and this is a slow process for debugging in general. What you need to do is add your own logging for your own platform of choice so you are not reliant on us for logs. Plus, that way, you can implement alerts, monitoring etc.
+Sort of. We will email you the logs for a failed sync through support but the number of log messages is limited and this is a slow process for debugging in general. What you need to do is add your own logging for your own platform of choice so you are not reliant on us for logs. Plus, that way, you can implement alerts, monitoring, etc.
 
-### Is it normal that for a sync, there is an upsert event followed by a truncate event?
-Yes definitely. This is most likely an initial sync where there is nothing but upsert operations, all followed by a truncate which is meant to (soft) delete any rows that may have existed prior to the initial sync starting to make sure they are all marked as deleted (since we cannot be sure the initial sync will necessarily overwrite them all). The "before timestamp" is key to the truncate operation so you don't just mark the entire table deleted. It should pick out the rows that existed prior to the sync starting (aka _fivetran_synced < "truncate before timestamp")
+### Is it normal that, for a sync, there is an upsert event followed by a truncate event?
+Yes, definitely. This is most likely an initial sync where there is nothing but upsert operations, all followed by a truncate, which is meant to (soft) delete any rows that may have existed prior to the initial sync starting. This is done to make sure all rows that may have existed prior to the initial sync are marked as deleted (since we cannot be sure the initial sync will necessarily overwrite them all). The "before timestamp" is key to the truncate operation so you don't just mark the entire table deleted. It should pick out the rows that existed prior to the sync starting (aka _fivetran_synced < "truncate before timestamp").
```

Comment:
@ediril Not sure about the format for '_fivetran_synced < "truncate before timestamp"' , should we use code `` backticks, at least, for `_fivetran_synced`?

Reply:
yes sure

Reply:
```suggestion
Yes, definitely. This is most likely an initial sync where there is nothing but upsert operations, all followed by a truncate, which is meant to (soft) delete any rows that may have existed prior to the initial sync starting. This is done to make sure all rows that may have existed prior to the initial sync are marked as deleted (since we cannot be sure the initial sync will necessarily overwrite them all). The "before timestamp" is key to the truncate operation so you don't just mark the entire table deleted. It should pick out the rows that existed prior to the sync starting (aka `_fivetran_synced` < "truncate before timestamp").
```